gradle/config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,278 @@
+<?xml version="1.0" ?>

```

Comment:
General comment: where possible, I'd err towards excluding any files where we haven't made Terra-specific exclusions or changes, and also prominently flagging where those changes have been made (e.g. with a <!-- Terra: {justification} --> comment above each exclusion)

Reply:
Whoops, forgot to commit those deletions.